packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -309,25 +310,22 @@ export async function sideloadAddIn(manifestPath: string, app?: OfficeApp, canPr
   if (isDesktop && app) {
     if (isSideloadingSupportedForDesktopHost(app)) {
       await registerAddIn(manifestPath);
-      if (app == OfficeApp.Outlook) {
+      if (app == OfficeApp.Outlook && isOutlookSideloadEnabled) {
```

Comment:
if the app is Outlook and sideload is not enabled, it shouldn't go to the else corresponding to the if when false. That would be non-outlook sideload. It should instead result in the error "Sideload is not supported for Outlook on Desktop."

Reply:
Ah stupid mistake - will fix this

Reply:
Moved this check up into isSideloadingSupportedForDesktopHost